src/pages/recipes.jsx

```diff
@@ -3,29 +3,27 @@ import { Link } from "gatsby";
 import styled from "styled-components";
 import PropTypes from "prop-types";
 
-
-import Layout from "../components/layout";
+import Layout from "../layout/main"; 
 import SEO from "../components/seo";
 
 const Wrapper = styled.div`
   margin: 50px 20px;
   padding: 10px;
 `;
-const Image= styled.img`
-width: 600px;
-height: 500px;
-`
+const Image = styled.img`
+  width: 600px;
+  height: 500px;
+`;
 const Recipes = ({ data }) => {
   const page = data.allContentfulRecipes.edges;
   return (
     <Layout>
       <SEO title="Recipes" />
       {page.map(({ node }) => {
-        const title = node.title || node.slug;
         return (
           <Wrapper>
             <h2 key={node.id}>
-              <Link to={`/${node.slug}`}>{title}</Link>
+              <Link to={`/${node.slug}`}>{node.title}</Link>
```

Comment:
now that you have a page for each recipe, may be worth updating this page a bit more to only be a listof recipetitles,like a directory, rather than listing all the full articles